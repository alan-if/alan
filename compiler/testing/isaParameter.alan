Syntax v = v (p).
The x Isa object
  Verb v
    Does
      If p Isa p Then ""
      End If.
  End Verb.
End The x.

The l Isa location
End The l.

Start At l.
