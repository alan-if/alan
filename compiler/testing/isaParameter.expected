########## isaParameter ##########


isaParameter.alan

    5.        If p Isa p Then ""
=====>                 1

  *1*   319 E : Identifier 'p' is not a Class.


        1 error(s).
        No detected warnings.
        1 informational message(s).

